### PR TITLE
add difficulty select

### PR DESCRIPTION
**Pull request by nikitakukharchuk**
Add difficulty select for questions on "Create test" page

### CODE REVIEW

File: src/components/new-test/QuestionBlock.tsx [hunk 1]
```diff
@@ -5,15 +5,22 @@ import { SuperSelect } from './SuperSelect';
 import { SuperInput } from './SuperInput';
 import { Timer } from './Timer';
 
+
 type QuestionBlockType = {
   value: string;
+  difficultyValue: string;
```
Comment (gitkoshelew) [inline src/components/new-test/QuestionBlock.tsx]: i think just 'difficulty' would be enough. an it should be Enum type

File: src/components/new-test/QuestionBlock.tsx [hunk 2]
```diff
@@ -5,15 +5,22 @@ import { SuperSelect } from './SuperSelect';
...
   handleTypeChange: (event: SelectChangeEvent) => void;
+  handleDifficultyChange: (event: SelectChangeEvent) => void;
```
Comment (gitkoshelew) [inline src/components/new-test/QuestionBlock.tsx]: please, change naming here. props handlers should start with 'on' and internal handlers with 'handle'
here is the topic https://jaketrent.com/post/naming-event-handlers-react/

File: src/components/new-test/QuestionBlock.tsx [hunk 3]
```diff
@@ -5,15 +5,22 @@ import { SuperSelect } from './SuperSelect';
...
   items: string[];
+  difficultyItems: string[];
 };
 
 export const QuestionBlock = ({
   value,
+  difficultyValue,
   handleTypeChange,
+  difficultyItems,
```
Comment (gitkoshelew) [inline src/components/new-test/QuestionBlock.tsx]: looks like you don't need drill difficulty Items throw all components. Just take them from the store

**Review by gitkoshelew — DISMISSED** [review]

**Review by gitkoshelew — DISMISSED** [review]

File: src/components/new-test/QuestionBlock.tsx (hunk 3 above)
Comment (nikitakukharchuk) [reply src/components/new-test/QuestionBlock.tsx]: use useAppSelector to resolve this task

File: src/components/new-test/QuestionBlock.tsx (hunk 2 above)
Comment (nikitakukharchuk) [reply src/components/new-test/QuestionBlock.tsx]: fix naming

File: src/components/new-test/QuestionBlock.tsx (hunk 1 above)
Comment (nikitakukharchuk) [reply src/components/new-test/QuestionBlock.tsx]: fix naming and add enum type

**Review by gitkoshelew — APPROVED** [review]

**Review by visiteur85 — APPROVED** [review]

**Pull request merged**